### ISSUE TITLE
When I click the timestamp, it just generate more new note pages.

### ISSUE THREAD

**Issue by sherrrrryx** [aidenlx/media-extended#437]
I saw others' tutorial videos about this plugin, in videos no matter how many times they click timestamp key, it generates timestamps on the same page, but mine keeps generating pages each timestamp. Did I set something wrong?

**Comment by Ovalt1ne** [comment]
You can set a **shortcut key** for "Media Extended: Take timestamp", don't use the icon in the player interface, that will only create a media note page. (Hope this solves your problem.)

**Comment by jamezchard** [comment]
I have encountered this issue as well. As @Ovalt1ne mentioned, you can use the "take timestamp" command instead of "take timestamp in media note".

However, this is more of a workaround. I believe the "take timestamp in media note" command doesn't work as expected.

**Comment by PlayerMiller109** [comment]
> I believe the "take timestamp in media note" command doesn't work as expected.

It seems that the author indeed designed in this way. You can check the explanation of the command here: https://github.com/PKM-er/media-extended/issues/432#issuecomment-2464470505

**Comment by victorwoo** [comment]
Yes, I understand, thank you. However, such a design does indeed confuse newcomers.

**Comment by ProfMaged** [comment]
> > I believe the "take timestamp in media note" command doesn't work as expected.
> 
> It seems that the author indeed designed in this way. You can check the explanation of the command here: [#432 (comment)](https://github.com/PKM-er/media-extended/issues/432#issuecomment-2464470505)

Actually no, because if you click more options and open media note, it will still open a new note. so it clearly doesn't work as intended. However, I found a temporary solution, If you just disable and enable Media Extended it does work properly. But when you restart Obsidian the bug is back and you have to disable and enable every time you restart obsidian and use the plugin. anyway, I just prefer to do other shortcuts with Commander for (take timestamp) and (save screenshot) until the bug is solved.

![Image](https://github.com/user-attachments/assets/51fecd1f-0b55-40fd-ab31-65078918e90a)

**Comment by aidenlx** [comment]
Fixed in v4.
